client/http-client/src/transport.rs

```diff
@@ -71,9 +55,10 @@ impl HttpTransportClient {
 		}
 
 		let client = match target.scheme_str() {
-			Some("http") => HyperClient::Http(Client::new()),
-			#[cfg(feature = "tls")]
-			Some("https") => {
+			#[cfg(not(feature = "__tls"))]
+			Some("http") => Client::new(),
+			#[cfg(all(feature = "__tls", feature = "native-tls", feature = "webpki-tls"))]
```

Comment:
Maybe I'm misunderstanding but this looks like it requires _both_ of those `-tls` features to be enabled (`all`)?

Reply:
Ooohhh no ok, you handle the individual oens below, got it!